### ISSUE TITLE
Missing resource exception in MusicController if no Midi output device set

### ISSUE THREAD

**Issue by hajdam** [jjazzboss/JJazzLab-X#259]
JJazzLab 3.0.2a on Fedora Linux, Java 11 OpenJDK

```
java.util.MissingResourceException: Can't find resource for bundle java.util.PropertyResourceBundle, key ERR_NoMidiOutputDeviceSet
	at java.base/java.util.ResourceBundle.getObject(Unknown Source)
	at java.base/java.util.ResourceBundle.getString(Unknown Source)
	at org.jjazz.util.api.ResUtil.getString(ResUtil.java:52)
	at org.jjazz.musiccontrol.api.MusicController.checkMidi(MusicController.java:1227)
	at org.jjazz.musiccontrol.api.MusicController.play(MusicController.java:395)
	at org.jjazz.ui.musiccontrolactions.Play.setSelected(Play.java:176)
	at org.jjazz.ui.musiccontrolactions.Play.actionPerformed(Play.java:113)
	at org.jjazz.ui.flatcomponents.api.FlatToggleButton.buttonClicked(FlatToggleButton.java:253)
	at org.jjazz.ui.flatcomponents.api.FlatButton$1.mouseClicked(FlatButton.java:97)
	at java.desktop/java.awt.AWTEventMulticaster.mouseClicked(Unknown Source)
	at java.desktop/java.awt.AWTEventMulticaster.mouseClicked(Unknown Source)
	at java.desktop/java.awt.Component.processMouseEvent(Unknown Source)
	at java.desktop/javax.swing.JComponent.processMouseEvent(Unknown Source)
	at java.desktop/java.awt.Component.processEvent(Unknown Source)
	at java.desktop/java.awt.Container.processEvent(Unknown Source)
	at java.desktop/java.awt.Component.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.Component.dispatchEvent(Unknown Source)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Unknown Source)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Unknown Source)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Unknown Source)
	at java.desktop/java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.Window.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.Component.dispatchEvent(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.desktop/java.awt.EventQueue$5.run(Unknown Source)
	at java.desktop/java.awt.EventQueue$5.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEvent(Unknown Source)
[catch] at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:136)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.run(Unknown Source)

```

**Comment by jjazzboss** [comment]
Hi Hajdam, welcome back, long time no see!
I'll fix this, it's just a missing string in a property file.

**Comment by jjazzboss** [comment]
Fixed in 3.0.3
